docs/contracts/v2/guides/staking/02-stake.md

```diff
@@ -0,0 +1,71 @@
+---
+id: "stake"
+sidebar_position: 2
+title: "Stake"
+---
+
+The `stake` function takes `tokenId` as input, and stake the stream by transferring the Sablier NFT from the user's
+wallet to the staking contract.
+
+```solidity
+function stake(uint256 tokenId) external {
```

Comment:
Let's rename this to `streamId`. Related:

https://github.com/sablier-labs/examples/pull/41